Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ContentGraph.php

```diff
@@ -126,76 +130,47 @@ public function findRootNodeAggregateByType(
             ));
         }
 
-        $rootNodeAggregate = $rootNodeAggregates->first();
-
-        if ($rootNodeAggregate === null) {
+        if (!$rootNodeAggregates->first()) {
             throw RootNodeAggregateDoesNotExist::butWasExpectedTo($nodeTypeName);
         }
 
-        return $rootNodeAggregate;
+        return $rootNodeAggregates->first();
```

Comment:
Why did you change this? IMO the former code was slightly better because it does not re-fetch the first result which (in theory) could be changed in the meantime

Reply:
Good question, why did I change this^^ Lets change it back, I do not remember and looking at it makes no sense to me either.